ostorlab.yaml

```diff
@@ -88,6 +88,10 @@ args:
     type: "string"
     description: "List of ports to scan."
     value: "0-65535"
+  - name: "tcp_syn_ping_ports"
+    type: "string"
+    description: "List of ports to use for host discovery (-PS)."
+    value: "21-22-25-53-68-68-80-110-123-143-443-465-631-993-995-3306-3389-8080"
```

Comment:
Shouldn't they be separated by ,?

Reply:
Nmap accept them separated by -, same pattren we have for ports.

https://nmap.org/book/man-host-discovery.html#:~:text=%2DPS22%2D25%2C80%2C113%2C1050%2C35000.

Reply:
Yes, we use `-` for ranges not when specifying a particular port.